src/main/scala/uk/gov/hmrc/testreport/model/PlatformExclusionRules.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.testreport.model
+
+object PlatformExclusionRules {
+
+  object GovUkBackLink
+      extends PlatformExclusionRule(
+        maybeHtmlRegex = Some(RegexPattern("""<a .*class="govuk-back-link.*</a>""")),
```

Comment:
I think it -could- -maybe- be a footgun to not be checking it's for the rule we mean when matching on html regex

more blast radius at least if you get the regex wrong

we don't store the ruleid in axeviolation at the moment, but we could, we wouldn't have to check the rule was what we think by looking at the help text via a regex like was done with old accessibilty assessment

but probably a future thing, since we're probably ok here - unless anyone has seen other legitimate issues with back link or skip link in services reports

Reply:
Agree, it seems quite brittle as it stands - feels like if we're going to change to use rule id we should maybe do it now?

Reply:
agreed to leave as-is for now, then if teams come to us with false positives due to the way they're constructing their HTML not matching the existing regexes we can revisit and maybe combine a less-specific regex with the axe violation id.